packages/jest-runtime/src/index.js

```diff
@@ -670,6 +671,7 @@ class Runtime {
         // $FlowFixMe
         (localModule.require: LocalModuleRequire),
       ), // jest object
+      ...extraGlobals.map(globalVariable => global[globalVariable]),
```

Comment:
```suggestion
      ...extraGlobals.map(globalVariable => this._environment.global[globalVariable])
```

can you test if this works? If so, we should use that instead.